src/compiler/AST/Typed.elm

```diff
@@ -196,3 +213,18 @@ recursiveChildren fn ( expr, _ ) =
 
         Tuple3 e1 e2 e3 ->
             fn e1 ++ fn e2 ++ fn e3
+
+
+mapExpr : (a -> b) -> Located ( a, c ) -> Located ( b, c )
```

Comment:
Let's make the types specific to document it better.
`mapExpr : (Expr_ -> Expr_) -> Located Expr -> Located Expr`